perf_dashboard/Makefile

```diff
@@ -22,7 +22,7 @@ push:
 	@docker push "$(IMG_VERSION)"
 
 deploy_only: get-cluster-credentials
-	@helm template --name "perf-dashboard" --set version="$(VERSION)" deploy/perf-dashboard | kubectl apply -f -
+	@helm install "perf-dashboard" deploy/perf-dashboard --set version="$(VERSION)" | kubectl apply -f -
```

Comment:
`install` and `kubectl apply` are mutually exclusive. We should only use one or the other. Probably `helm template`